### DISCUSSION TITLE
CG visualisation

### DISCUSSION THREAD

**Discussion by csbrasnett** [nglviewer/nglview#1119] [General]
I'm looking into how cg (specifically Martini) simulations can be visualised with different software. I'm aware of #818 already as a highlight of this topic for nglview specifically. What I'd really like to know but can't work out is _how_ the bond guessing works. From what I can see, I think it's some kind of distance measure, because I'm getting things like an ARG side chain linking everything in a triangle, rather than a linear BB-SC1-SC2 as is actually implemented in the topology. 

Additionally, would there be a way of loading a topology file (eg. *.top, *.itp, whatever) into the visualisation to additionally see extra bonded networks. Martini proteins get held together with elastic networks, which are then much larger bonds than could be 'guessed'. Is there a way of adding extra bonds between points in nglview?

**Comment by hainm** [comment]
Dear @csbrasnett Unfortunately, I don't have answers for all of your questions.
